fs/procfs/fs_procfs.c

```diff
@@ -339,6 +339,32 @@ static void procfs_enum(FAR struct tcb_s *tcb, FAR void *arg)
   dir->pid[index] = tcb->pid;
   dir->base.nentries = index + 1;
 }
+
+/****************************************************************************
+ * Name: procfs_sort_pid
+ ****************************************************************************/
+
+static void procfs_sort_pid(FAR struct procfs_level0_s *level0)
+{
+  pid_t pid;
+  int i;
+  int j;
+
+  /* Sort the process id by Bubble */
+
```

Comment:
Maybe add a "FIXME: improve searching algorithm" here.

Reply:
There is one:
https://github.com/apache/nuttx/blob/master/include/stdlib.h#L276-L277
but @anchao avoid to use it intentionally for simplity.

Reply:
Done